### DISCUSSION TITLE
Deployment Error - What are the Location and Principal ID parameters for deployment?

### DISCUSSION THREAD

**Discussion by isinghrana** [Azure-Samples/chat-with-your-data-solution-accelerator#920] [Q&A] [answered]
I am using the Deploy button on the Readme but deployment is giving errors without much details so I am stuck. 
Is it some of the parameters?

There is a Location parameter on Deployment screen which I am not clear about, since Region is already specific does it need to be the same value as region or something else?
Unclear on Principal ID parameter? If Auth Type selection is Keys can I leave this blank?

Not sure if the above parameters are the issue or is it something else. I get different error every time I try to deploy, I tried different regions as well. I think accelerator is very useful but it would be good get some more documentation on the deployment, troubleshooting, etc. I am re-deploying and will add the deployment error messages on this thread in a little bit, but mostly these messages are cryptic like Internal Server Error, Resource Group not found when the Resource Group is created as part of the deployment.

Any guidance will be greatly appreciated.

Thanks,
Inder

**Comment by isinghrana** [comment]
Deployment Error 
Computer Vision deployment error, error message - "{"code":"ResourceGroupNotFound","message":"Resource group 'rg-israoai9' could not be found."}"
The resource group mentioned is the one which is being created by this deployment.

**Comment by isinghrana** [comment, marked as answer]
I saw the following commit https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/pull/919 which had the comment Computer Vision Fix and re-deployed so that resolved the Compute Vision Resource Group error - https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/pull/919

**Reply by ross-p-smith** [reply]
Sorry that you found this bug @isinghrana - we will update the deployment documents
